omegaconf/typing.py

```diff
@@ -0,0 +1,10 @@
+import sys
+
+if sys.version_info >= (3, 10):
+    from typing import TypeAlias
+else:
+    from typing_extensions import TypeAlias
+
+from .vendor.antlr4.ParserRuleContext import ParserRuleContext
+
+Antlr4ParserRuleContext: TypeAlias = ParserRuleContext  # type: ignore[valid-type]
```

Comment:
What is the purpose of this file, and in particular why did you have to "rename" ParserRuleContext  to Antlr4ParserRuleContext?

Reply:
Gotcha.
I wonder if MyPy has a way to globally exclude a specific type...

If not, 
Can you add a comment explaining the alias?
Also, I think you could keep the name the same, just import it from a different place.